### PR TITLE
RUN-29

### PR DESCRIPTION
**Pull request by sam-kakengi**
Added the running app, created the two database models RunActivity and Route, included migrations and entrypoint.sh files.

### CODE REVIEW

**Review by Dimwiddle — CHANGES_REQUESTED** [review]
The general code looks @sam-kakengi 

However there's a few bits here:

- The RUN-29 ticket only includes creating the models.py, migrations and creating of an entrypoint.sh script. 
- Even though some of the API code looks OK, it's not part of this ticket. And actually some of the running  endpoints  are not according to the requirements. So we don't want to get ahead of what needs to be done just yet.

**To Do:**
1. Create the migrations of the current `models.py`
2. Add an `entrypoint.sh` to the `backend/` - to include the migration commands e.g. python manage.my migrate)
3. Clear the API related work files (for now) in `running` app: `views.py`, `urls.py`, `serializers.py`

**Comment by sam-kakengi** [comment]
Ah yes I forgot to remove those, was playing around to see if it was all setup with those extra steps will remove them. Also there is already a entrypoint.sh file in backend/ within the main and develop repositories that contains migration commands?

File: backend/backend/urls.py [hunk 1]
```diff
@@ -15,8 +15,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/running/', include('running.urls')),
```
Comment (Dimwiddle) [inline backend/backend/urls.py:22]: Can you reverse this as well?

**Review by Dimwiddle — CHANGES_REQUESTED** [review]
Looking good. Just one more fix then good to merge in.

No need to request another review after this. Just go ahead and merge 👍 @sam-kakengi

**Pull request merged**